guide/language-concepts/classes.markdown

```diff
@@ -225,8 +240,10 @@ how to make complex decisions in CFEngine, with readable results. It is like
 defining aliases for class combinations. Such class 'aliases' may be specified 
 in any kind of bundle.
 
-Note that whitespace is *not* allowed between operators, so something like
-`a . b`, though perhaps more readable, will be rejected by the CFEngine parser.
+Note that whitespace is **only** allowed between operators when the
+expression is quoted as a string, so something like `a . b`, though
+perhaps more readable, will be rejected by the CFEngine parser. You
+need to use `"a . b"` instead.
```

Comment:
Didn't we change this so that it *is* allowed?

Can also remove the "when quoted as strings" part above in that case.